server/src/test/java/org/opensearch/action/search/SearchRequestTests.java

```diff
@@ -134,25 +132,6 @@ public void testRandomVersionSerialization() throws IOException {
         }
     }
 
-    public void testReadFromPre6_7_0() throws IOException {
-        String msg = "AAEBBWluZGV4AAAAAQACAAAA/////w8AAAAAAAAA/////w8AAAAAAAACAAAAAAABAAMCBAUBAAKABACAAQIAAA==";
-        try (StreamInput in = StreamInput.wrap(Base64.getDecoder().decode(msg))) {
-            in.setVersion(
-                VersionUtils.randomVersionBetween(
-                    random(),
-                    LegacyESVersion.V_6_4_0,
```

Comment:
Would this test work for versions between `V_6_5_0` and `V_6_7_0`?

Reply:
This is no longer required, as we are removing all 6.x constants eventually in subsequent PRs. See #1674

Reply:
Makes sense!